udata/core/spatial/api.py

```diff
@@ -32,6 +35,35 @@
     'size', type=int, help='The amount of suggestion to fetch',
     location='args', default=10)
 
+dataset_parser = api.parser()
+dataset_parser.add_argument(
+    'dynamic', type=bool, help='Append dynamic datasets',
```

Comment:
From where do I import that? I can't find it in [that doc](http://flask-restplus.readthedocs.io/en/stable/parsing.html).